src/core/mod.rs

```diff
@@ -95,6 +122,7 @@ impl std::ops::AddAssign for Size {
 
 #[cfg(test)]
 mod tests {
+    const ZERO: Size = Size::new(0, 0);
```

Comment:
I can see why you moved this, but I think it should still be accessible from the rest of the code (in this case as `Size::ZERO`):
```rust
impl Size {
     pub const ZERO = ...;

     ...
}

Reply:
I moved this into tests because it only used in tests.